tests/test_version.py

```diff
@@ -32,4 +32,4 @@
 
 
 def test_version():
-    assert __version__ == '0.1.0.dev7'
+    assert __version__ == '0.1.1'
```

Comment:
we should delete this test... it's pointless and always wrong when we do an update.

Reply:
I tend to agree. What is the test trying to catch in the first place?

Reply:
I have no idea what this was catching. I bet it was from a Cookiecutter tempater @aviveiros11 ...

Reply:
👀 - @nllong I think the mold might already be getting to your head.  But yeah I'll get right on it.  Throw it on my schedule for - say - this Fall?

Reply:
haha, signing you up for the fall! Alfalfa is going to be huge, you should jump on the bandwagon!